romana-install/roles/romana/install-agent/templates/config/romana.yaml

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
tested it and it does work, we use viper which in turn uses go-yaml/yaml which does parse this correctly.